semgrep-core/CLI/Main.ml

```diff
@@ -1213,6 +1213,8 @@ let all_actions () = [
   Common.mk_action_2_arg Test_synthesizing.expr_at_range;
   "-synthesize_patterns", " <l:c-l:c> <file>",
   Common.mk_action_2_arg Test_synthesizing.synthesize_patterns;
+  "-generate_patterns", " <file>",
```

Comment:
it also takes a list of ranges no? so the comment string should be <l:c-l:c>+ <file>